comms/dht/src/store_forward/service.rs

```diff
@@ -437,7 +438,17 @@ impl StoreAndForwardService {
             Anonymous => db.find_anonymous_messages(query.since, limit).await?,
         };
 
-        Ok(messages)
+        // Reduce the overall amount of messages returned to the window, should be close enough since this is also
+        // checked when storing or propagating the messages.
+        let validity_window = chrono::Duration::from_std(self.config.saf_msg_validity).unwrap();
```

Comment:
This is the only part of the PR I disagree with. The storing node already has rules around what it stores. Also, this gives two meanings to `saf_msg_validity`: 1. The expiry on sent messages and 2. The max value of expiry of stored messages. This is confusing about what value is being used.

I would suggest remove this change to the file completely as it's already handled by existing code

Reply:
This has been removed.